internal/sirius/get_deputy_clients.go

```diff
@@ -0,0 +1,185 @@
+package sirius
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sort"
+	"strings"
+	"time"
+)
+
+type apiOrder struct {
+	OrderStatus struct {
+		Label string `json:"label"`
+	}
+	LatestSupervisionLevel *struct {
+		SupervisionLevel struct {
+			Label string `json:"label"`
+		}
+	}
+	OrderDate string `json:"orderDate"`
+}
+
+type apiOrders []apiOrder
+
+type apiClients struct {
+	Clients []struct {
+		ClientId            int    `json:"id"`
+		Firstname           string `json:"firstname"`
+		Surname             string `json:"surname"`
+		CourtRef            string `json:"caseRecNumber"`
+		RiskScore           int    `json:"riskScore"`
+		ClientAccommodation struct {
+			Label string `json:"label"`
+		}
+		Orders apiOrders
+	} `json:"persons"`
+}
+
+type Order struct {
+	OrderStatus      string
+	SupervisionLevel string
+	OrderDate        time.Time
+}
+
+type Orders []Order
+
+type DeputyClient struct {
+	ClientId          int
+	Firstname         string
+	Surname           string
+	CourtRef          string
+	RiskScore         int
+	AccommodationType string
+	OrderStatus       string
+	SupervisionLevel  string
+}
+
+type DeputyClientDetails []DeputyClient
+
+func (c *Client) GetDeputyClients(ctx Context, deputyId int) (DeputyClientDetails, error) {
+	req, err := c.newRequest(ctx, http.MethodGet, fmt.Sprintf("/api/v1/deputies/%d/clients", deputyId), nil)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := c.http.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode == http.StatusUnauthorized {
+		return nil, ErrUnauthorized
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		return nil, newStatusError(resp)
+	}
+
+	var v apiClients
+	if err = json.NewDecoder(resp.Body).Decode(&v); err != nil {
+		return nil, err
+	}
+
+	var clients DeputyClientDetails
+	for _, t := range v.Clients {
+		orders := RestructureOrders(t.Orders)
+		if len(orders) > 0 {
+			var client = DeputyClient{
+				ClientId:          t.ClientId,
+				Firstname:         t.Firstname,
+				Surname:           t.Surname,
+				CourtRef:          t.CourtRef,
+				RiskScore:         t.RiskScore,
+				AccommodationType: t.ClientAccommodation.Label,
+				OrderStatus:       GetOrderStatus(orders),
+				SupervisionLevel:  GetMostRecentSupervisionLevel(orders),
+			}
+			clients = append(clients, client)
+		}
+	}
+	alphabeticalSort(clients)
+	return clients, err
+}
+
+func GetOrderStatus(orders Orders) string {
```

Comment:
Functions that aren't used outside of their package should start with a lower case letter. Capitals make it public.

Reply:
Also, if the comment below is a description of what the function does, it should be written as a docstring above the function declaration, e.g. 
```
/**
  * GetOrderStatus returns the status of the oldest active order for a client.
  * If there isn’t one, the status of the oldest order.
  ** /
  func getOrderStatus(orders Orders) string {
```